### PR TITLE
Removing admin-consent related changes in registration script as it is no more required

### PR DESCRIPTION
**Pull request by bniranjanbhat**
## Description

HCI Cloud services does not require Roles to be part of the AAD token claim anymore.  This in turn avoids admin-consent during registration. The changes in this PR removes all the code to do with Admin Consent changes.
Other misc change:
1. With the latest GA version of Az.resource PS module, registration fails when custom Azure Environments (Canary in our case) is used. Turns out for custom environments, we need to configure MicrosoftGraphUrl and MicrosoftGraphEndpointResourceId separately while creating it.
2. Removing cluster name for clusterScheduledTask related operations since it is already in cluster's context
## Checklist

- [ X ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ X ] The title of the PR is clear and informative
- [ X ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [X ] If applicable, the changes made in the PR have proper test coverage
- [ X] For public API changes to cmdlets:
    - [X ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ X] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/StackHCI/Az.StackHCI.psm1 [hunk 1]
```diff
@@ -912,7 +691,7 @@ param(
         $azEnv = (Get-AzEnvironment -Name $AzureCloud)
         $azEnv.Name = $ConnectAzAccountEnvironmentName
         $azEnv.ResourceManagerUrl = ('https://{0}.management.azure.com/' -f $Region)
-        $azEnv | Add-AzEnvironment | Out-Null
+        $azEnv | Add-AzEnvironment -MicrosoftGraphEndpointResourceId "https://graph.microsoft.com" -MicrosoftGraphUrl "https://graph.microsoft.com" | Out-Null
```
Comment (sugatt) [inline src/StackHCI/Az.StackHCI.psm1:694]: why is this needed?

**Review by sugatt — APPROVED** [review]

**Comment by NoriZC** [comment]
Hi @bniranjanbhat, as this draft PR hasn't been updated for long, could you please check and close it? Thanks!

**Comment by muthus-msft** [comment]
Adding @Niranjan ***@***.***>
